assets/stylesheets/layout/_main.scss

```diff
@@ -275,54 +280,58 @@ html.iframed {
 $sidebar-width-max: 272px;
 $sidebar-width-min: 228px;
 
+.wp {
+	display: flex;
+	flex-direction: column;
+	position: fixed;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+}
+
 .wp-content {
-	@include clear-fix;
 	position: relative;
-	margin: 47px 0 0 0;
-	padding: 32px 32px 32px ( $sidebar-width-max + 32px );
+	display: flex;
 	box-sizing: border-box;
 	overflow: hidden;
+	flex: 1 1 auto;
+}
 
-	.has-no-sidebar & {
-		padding-left: 32px;
-	}
+#primary {
```

Comment:
We don't want to target these elements in styling. Is there another way? If anything it should use `.wp-primary` instead.